website/content/en/preview/concepts/disruption.md

```diff
@@ -165,33 +135,49 @@ Read the [Drift Design](https://github.com/aws/karpenter-core/blob/main/designs/
 __Behavioral Fields__
 - Weight                      
 - Limits                      
-- Consolidation               
-- TTLSecondsUntilExpired      
-- TTLSecondsAfterEmpty  
+- ConsolidationPolicy               
+- ConsolidateAfter      
+- ExpireAfter  
 ---      
-#### AWSNodeTemplate
-| Fields                     | One-way | Two-way  |
-|----------------------------|  :---:  |  :---:   |
-| Subnet Selector            |         |    x     |
-| Security Group Selector    |         |    x     |
-| Instance Profile           |    x    |          |
-| AMI Family                 |    x    |          |
-| AMI Selector               |         |    x     |
-| UserData                   |    x    |          |
-| Tags                       |    x    |          |
-| Metadata Options           |    x    |          |
-| Block Device Mappings      |    x    |          |
-| Detailed Monitoring        |    x    |          |
-
-To enable the drift feature flag, refer to the [Settings Feature Gates]({{<ref "./settings#feature-gates" >}}).
-
-Karpenter will add `MachineDrifted` status condition on the machines if the machine is drifted, and does not have the status condition,
-
-Karpenter will remove the  `MachineDrifted` status condition for the following these scenarios:
-1. The `featureGates.driftEnabled` is not enabled but the machine is drifted, karpenter will remove the status condition.
-2. The machine isn't drifted, but has the status condition, karpenter will remove it.
-
-If the node is marked as drifted by another controller, karpenter will do nothing.
+##### EC2NodeClass
+| Fields                        | One-way | Two-way |
+|-------------------------------|:-------:|:-------:|
+| Subnet Selector Terms         |         |    x    |
+| Security Group Selector Terms |         |    x    |
+| AMI Family                    |    x    |         |
+| AMI Selector     Terms        |         |    x    |
```

Comment:
These look like odd spaces in the raw, is it trimmed down in the rendered version?

Reply:
Whoops, good catch 👍